js/config/scoring.js

```diff
@@ -14,30 +14,40 @@ YoastSEO.AnalyzerScoring = function( i18n ) {
                 {
                     min: 300,
                     score: 9,
+
+                    /* translators: %1$d expands to the amount of words in the text, %2$s to the recommended minimum of words */
```

Comment:
"%1$d expands to the amount of words..." --> "%1$d expands to the number of words..."